src/VariableInput/VariableInput.scss

```
@@ -0,0 +1,70 @@
+@import '~draft-js/dist/Draft';
+
+@import '../common';
+@import '../typography-helpers';
+
+$minWidth: 378px;
+$minHeight: 30px;
+$backgroundColor: $D80;
+$border: 1px solid $B30;
+$borderRadius: 6px;
+$padding: 0 7px;
+$margin: 2px 0;
+
+.root {
+  position: relative;
+  text-align: left;
+  min-height: $minHeight;
+  min-width: $minWidth;
+  background-color: $backgroundColor;
+  border-radius: $borderRadius;
+
+  :global(.public-DraftEditorPlaceholder-root) {
+    @include Text($weight: normal, $size: medium, $light: true, $secondary: true);
+
+    margin: $margin;
+    position: absolute;
+    pointer-events: none;
+    top: 4px;
+    left: 10px;
+  }
+
+  :global(.public-DraftEditor-content) {
+    background-color: $backgroundColor;
+    min-height: $minHeight;
+    max-height: var(--max-height);
+    padding: $padding;
+    border: $border;
+    border-radius: $borderRadius;
+    overflow: auto;
+    box-sizing: border-box;
+    line-height: 30px;
+
+    @include Text($weight: thin, $size: medium);
+
+    &:hover {
+      background-color: $B50;
+      cursor: pointer;
+    }
+
+    &:focus {
+      background-color: $D80;
+      cursor: text;
+      border-color: $B10;
+      box-shadow: 0 0 0 3px $F00;
+    }
+  }
+}
+.tag-wrapper {
+  line-height: initial;
+  padding-left: 2px;
+  padding-right: 2px;
+  > span{
```

Let's replace it with a dedicated class selector :)

Like:
```suggestion
  > .tag {
```